velox/docs/functions/spark/json.rst

```diff
@@ -84,3 +84,4 @@ JSON Functions
         SELECT json_object_keys(''); -- NULL
         SELECT json_object_keys(1); -- NULL
         SELECT json_object_keys('"hello"'); -- NULL
+        SELECT json_object_keys("invalid json"); -- NULL
```

Comment:
Met the some errors when running this and above examples in Spark. Could you clarify?